WebApiCoreSeed.WebApi/Controllers/UserController.cs

```diff
@@ -38,7 +55,12 @@ public UserController(IUserService userService)
             return Ok(user);
         }
 
-        // POST api/users
+        /// <summary>
+        /// Creates a new user
+        /// </summary>
+        /// <param name="user" cref="UserDto">User model</param>
+        /// <response code="204">User created</response>
+        /// <response code="404">User could not be created</response>
         [HttpPost]
         [ValidateModel]
         public async Task<IActionResult> Create([FromBody]UserDto user)

```

Comment:
In this case, you are not specifying what output is produced, Does Swagger infer it based on XML comments?

Reply:
yes, also, note that is a 204, with the message should be ok.

Reply:
Ok, I saw that you are using _IncludeXmlComments_, it is impressive.

I understand that you are using XML comments in place of _ProducesResponseType_ because you do not have a model for these outputs, right? 

Is there not a way to do that using _ProducesResponseType_? (I mean, to always use the same way to define the outputs)

Reply:
you're right, it is incongruent using it both ways, let me see if it is a way to use only xml (so you are obligated to document the api)

Reply:
Personally, I prefer _ProducesResponseType_ because it allows using existent types.

Anyway, it is only a detail, your approach looks very clear beside this _inconsistence_.

Reply:
swashbuckle says that it is ok to use both, the attribute only defines the type of the outputs, when the xml comment just displays a description

![capture](https://user-images.githubusercontent.com/7729931/39781507-b72540b6-52e5-11e8-8adf-41d6e085959b.PNG)


[https://github.com/domaindrivendev/Swashbuckle.AspNetCore#include-descriptions-from-xml-comments](url)